JSON_for_IO-Link.yaml

```diff
@@ -4279,660 +4176,327 @@ components:
           "2476": false
           "2477": false
           "2478": false
-          "2479": false
-          
-    tracksStatisticsChannel:
+    
+    mastersPortsPairingPost:                                                               # TODO: IOLW
       properties:
-        trackNumber:
+        portPairing:
+          type: string
+          enum:
+            - Unpairing
+            - PairingUnique
+            - PairingButton
+      example:
+        WPortPairing: PairingUnique
+
+    mastersScanGet:                                                                 # TODO: IOLW
+      properties:
+        scanStatus:
+          type: string
+          enum:                    
+            - NoScanWasPerformed   
+            - ScanInProgress
+            - ScanEnded
+        scanResults:
+          type: array
+          properties:
+            slotType:
+              type: string
+              enum:
+                - "SSLOT"
+                - "DSLOT"
+            uniqueId:
+              type: string
+              minLength: 26
+              maxLength: 26
+            ioLinkRevision:
+              type: string
+              enum:
+                - '1.1'
+      example:
+        scanStatus: ScanEnded
+        scanResults:
+          - slotType: SSLOT
+            uniqueId: 03:78:00:00:01:32:50:60:46
+            ioLinkRevision: 1.1
+          - slotType: DSLOT
+            uniqueId: 03:78:00:00:01:32:50:60:47
+            ioLinkRevision: 1.1
+
+    mastersScanPost:                                                            # TODO: IOLW
```

Comment:
Warum hier kein Parameter mit einem Timeout-Wert?

Reply:
I would not recommend a timeout rather the time since the last scan.
@mrentsch65 What do you think about that?

Reply:
It's ok for me. I was just wondering that something might be missing. Thus when you start a scan this will go on forever?